backend/hostgw/hostgw.go

```diff
@@ -133,7 +137,7 @@ func (rb *HostgwBackend) handleSubnetEvents(batch []subnet.Event) {
 	for _, evt := range batch {
 		switch evt.Type {
 		case subnet.SubnetAdded:
-			log.Infof("Subnet added: %v via %v", evt.Lease.Subnet, evt.Lease.Attrs.PublicIP)
+			log.Infof("Subnet added: %v via %v", evt.Lease.Subnet, evt.Lease.Attrs.InterfaceIP)
```

Comment:
`InterfaceIP` is only used (read) in this backend now. And it's equal to `PublicIP` (because you check). Therefore you can go back to using `PublicIP` here and delete `InterfaceIP` from `LeaseAttrs`.